src/envoy/utils/grpc_transport.h

```diff
@@ -62,7 +62,7 @@ class GrpcTransport : public Grpc::TypedAsyncRequestCallbacks<ResponseType>,
 
   Grpc::AsyncClientPtr async_client_;
   ResponseType* response_;
-  const ::istio::mixer::v1::Attributes& forward_attributes_;
+  std::string serialized_forward_attributes_;
```

Comment:
I think you can use const &,  not need to copy the data

Reply:
Done